### DISCUSSION TITLE
CardioVascular use cases related resources with Imaging

### DISCUSSION THREAD

**Discussion by ghost** [synthetichealth/synthea#1125] [General]
I have generated a population based on Synthea using the "chf_*" modules. I am noticing that the resources that get generated in particular the ImageStudies are not linked to a DiagnosticReport resource that documents the findings. What am I missing here?

**Comment by jawalonoski** [comment]
You are not missing anything.

An `ImagingStudy` is conducted and recorded for every [ImagingStudy State](https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-States#imagingstudy) in a module.

A `DiagnosticReport` resource is only generated if:
-  The module explicitly defines a [DiagnosticReport State](https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-States#diagnosticreport)
- Or, if US Core is enabled (it is enabled by default), then a `DiagnosticReport` and `DocumentReference` are generated for each `Encounter` to capture simple template-driven clinical notes. 

We currently support US Core v4 (with a PR in-progress #1103), and only use the [v4/STU4 US Core DiagnosticReport Profile for Report and Note exchange](http://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-diagnosticreport-note.html) for generic clinical notes (not imaging results).

US Core v5 has the [US Core Observation Imaging Profile](http://hl7.org/fhir/us/core/StructureDefinition-us-core-observation-imaging.html) which could be used with the [US Core DiagnosticReport Profile for Report and Note Exchange ](http://hl7.org/fhir/us/core/StructureDefinition-us-core-diagnosticreport-note.html) to document the `ImagingStudy` findings, but we do not currently generate these automatically, nor do we currently support US Core v5.

Even if we did support US Core v5, the issue with generating these automatically is that Synthea modules do not currently explicitly define the results. So, we would need to change all the models to do so, or figure out a way to make clinically reasonable inferences on what those findings would be in the context of the module workflow. Which, I am sure you understand, is non-trivial.

Everything in FHIR is optional. Everything in a Synthea module is optional, and unless it is defined, it doesn't happen. We're happy to take pull requests.
